index.js

```diff
@@ -419,6 +419,52 @@
     return f(y, x);
   });
 
+  //# lift :: Applicative f => (a -> b) -> f a -> f b
+  //.
+  //. Takes a unary function and promotes it to a function which operates on
+  //. a functor.
```

Comment:
Nice! This is much better. We could shorten it a little:

_Promotes a unary function to a function which operates on a functor._

Also, should we say "an [Apply][1]" rather than "a functor"?


[1]: https://github.com/fantasyland/fantasy-land#apply